src/task3/services/userService.js

```diff
@@ -0,0 +1,32 @@
+class UserService {
+    constructor(userModel) {
+        this.model = userModel;
+    }
+    getUsers(page, limit) {
+        return new Promise((resolve, reject) => this.model.findAll({ offset: page * limit, limit, where: { isDeleted: false } }).then(users => {
```

Comment:
Consider creating a promise factory to reduce the boilerplate code in each handler.
Beware of the "this" context, since it might be implicitly lost withing the factory.